radarr/installation/multiple-instances.md

```diff
@@ -53,9 +53,9 @@ This guide will show you how to run multiple instances of Radarr on Windows usin
 1. Open a Command Prompt administrator window. (To run as an admin,
     right click on the Command Prompt icon and choose “Run as
     administrator.”)
-1. If Radarr is running, stop the service by running `nssm stop Radarr`
+2. If Radarr is running, stop the service by running `nssm stop Radarr`
```

Comment:
Keep as 1. Wiki numbers it when rendering

Reply:
Oh i had no idea!